### DISCUSSION TITLE
How to deal with low values and 0s in logged data

### DISCUSSION THREAD

**Discussion by shipmadison** [atsa-es/fish550-2023#15] [Q&A]
Hi all! 

For Lab 1, I'm working with pink salmon in the ruggerone_data. For the return totals, some of my values are very small (<1, or 0), which gives me a negative log value, or -infinity. I was thinking about ways to deal with this problem (outside of eliminating the problem region that had all the 0's) and initially added 1 to the returns as a smoothing factor prior to logging. However, this will disproportionally impact low returns, so this method doesn't seem appropriate. What is the best way to approach this problem? Thanks much!

**Comment by eeholmes** [comment]
Hi Madison,

For this lab, I would eliminate the problem region. Note pink salmon are on a strict 2-year cycle that don't overlap with each other. So you might want to model the even and odd years separately. If one of those are mainly 0s, then just model the cycle that is non-zero.

Eli

**Comment by shipmadison** [comment]
Ah! That makes a lot of sense! Thanks for the tip. I'll explore some data parsing and region elimination.
